rust/connlib/tunnel/src/control_protocol/client.rs

```diff
@@ -106,18 +106,22 @@ where
             &domain_response.as_ref().map(|d| d.domain.clone()),
         )?;
 
-        let mut peer: Peer<_, PacketTransformClient> =
-            Peer::new(ips.clone(), gateway_id, Default::default());
+        let mut resource_ids = HashSet::new();
+        resource_ids.insert(resource_id);
```

Comment:
```suggestion
        let resource_ids = HashSet::from([resource_id]);
```

Reply:
oh I didn't know you could write it like this, nice.